libs/platform/src/lib/approval-flow/tsconfig.json

```diff
@@ -21,6 +21,7 @@
   },
   "angularCompilerOptions": {
     "strictInjectionParameters": false,
-    "strictTemplates": false
+    "strictTemplates": false,
+    "fullTemplateTypeCheck": false
```

Comment:
Why this change?

Reply:
all this should be true, and it's in a roadmap
added this row so folks didn't forget to enable it later